### PR TITLE
Update yt

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [x] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

## What issue is being fixed?

https://github.com/AdguardTeam/AdguardFilters/issues/170786

### Add your comment and screenshots

Details are on Slack. As the anti-adb is only sent in www, limited to this.

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -154,6 +154,13 @@ youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', '
 youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', 'ytInitialPlayerResponse.playerAds', 'undefined')
 youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', 'playerResponse.adPlacements', 'undefined')
 !
+! Remove anti-adb
+!#if (adguard_app_windows || adguard_app_mac || adguard_app_android || adguard_ext_firefox)
+.com/watch?v=$replace=/"auxiliaryUi.*?e\}\}\}\,//,domain=www.youtube.com
+.com/watch?v=$replace=/("trackingParam":"kx_fmPxhoPZR)[-_0-9A-Za-z]{275}([-_0-9A-Za-z]{34}zRgkuMsNLBwOcCE59TDtslLKPQ-SS"\})/\$1\$2/,domain=www.youtube.com
+/youtubei/v*/player?key=$replace=/"auxiliaryUi.*?e\}\}\}\,//,domain=www.youtube.com
+/youtubei/v*/player?key=$replace=/("trackingParam":"kx_fmPxhoPZR)[-_0-9A-Za-z]{275}([-_0-9A-Za-z]{34}zRgkuMsNLBwOcCE59TDtslLKPQ-SS"\})/\$1\$2/,domain=www.youtube.com
```
Comment (Alex-302) [inline BaseFilter/sections/specific.txt:162]: Rules with `zRgkuMsNLBwOcCE59TDtslLKPQ` don't match on my end
https://regex101.com/r/gKZTp6/1

Comment (Yuki2718) [reply BaseFilter/sections/specific.txt:162]: Okay, will change to `[0-9A-Za-z]{9}BwOcCE59TDtslLKPQ`

**Comment by Alex-302** [comment]
@AdamWr @piquark6046 Could you please take a look? 
On my end rules applied, but without them end disabled scriptlets I can't trigger anti-adblock.

**Comment by AdamWr** [comment]
If I understand correctly, these rules are applied only if there is adblock detection (a real detection, not invoked by a special script or something like that).
If so, then I cannot check if it works or not, because there is no detection on my account, but I don't see any breakage with these rules.

**Comment by Alex-302** [comment]
Let's wait few days and check if the rules still match that parts.

**Comment by Yuki2718** [comment]
> Let's wait few days and check if the rules still match that parts.

Already a week passed, still matches. Their encoding doesn't change until they discover this and counter, only concern is the encoded text itself.
@AdamWr @piquark6046 Are `kx_fmPxhoPZR` and `BwOcCE59TDtslLKPQ-SS` present in your `responseContext.mainAppWebResponseContext.trackingParam`?

**Comment by Yuki2718** [comment]
Tested on the Windows app, works perfectly against real anti-adb.

**Comment by piquark6046** [comment]
https://github.com/AdguardTeam/AdguardFilters/pull/170936#issuecomment-1898171743
`responseContext.mainAppWebResponseContext.trackingParam` does not exist in my personal Google account.

**Comment by Yuki2718** [comment]
@piquark6046 You have to click some videos, then check `player?key=` requests in Network tab of devtools.

**Comment by AdamWr** [comment]
> Are `kx_fmPxhoPZR` and `BwOcCE59TDtslLKPQ-SS` present in your `responseContext.mainAppWebResponseContext.trackingParam`?

Yes, I see it, but it looks like that rules are not applied on my end, so I guess that this `trackingParam` is different when detections occur.

**Comment by piquark6046** [comment]
> Are kx_fmPxhoPZR and BwOcCE59TDtslLKPQ-SS present in your responseContext.mainAppWebResponseContext.trackingParam?

OK. I found `kx_fmPxhoPZR` and `BwOcCE59TDtslLKPQ-SS` without any account.

**Comment by Yuki2718** [comment]
> Yes, I see it, but it looks like that rules are not applied on my end, so I guess that this trackingParam is different when detections occur.

You mean not applied as the rule targets longer strings? That's expected - they add strings about anti-adb to `trackingParam` if they send anti-adb. The original `trackingParam` is still preserved, but interesingly it is in the tail dispite in the head in decoded text.

**Comment by AdamWr** [comment]
>You mean not applied as the rule targets longer strings?

Yes, that's what I meant.

**Comment by Yuki2718** [comment]
I found shorter strings in requests saved three monts ago. Will adjust the length. Note, as already mentioned, the rules won't apply usually - they match only if the warning is sent. If any problem arises, it must be only when the case otherwise user see anti-adb and we can tell such reports from characteristic error in the player.

**Pull request merged**